lib/rules/declaration-property-value-keyword-no-deprecated/index.cjs

```diff
@@ -0,0 +1,178 @@
+// NOTICE: This file is generated by Rollup. To modify it,
+// please instead edit the ESM counterpart and rebuild with Rollup (npm run build).
+'use strict';
+
+const valueParser = require('postcss-value-parser');
+const validateTypes = require('../../utils/validateTypes.cjs');
+const nodeFieldIndices = require('../../utils/nodeFieldIndices.cjs');
+const getDeclarationValue = require('../../utils/getDeclarationValue.cjs');
+const isStandardSyntaxDeclaration = require('../../utils/isStandardSyntaxDeclaration.cjs');
+const optionsMatches = require('../../utils/optionsMatches.cjs');
+const report = require('../../utils/report.cjs');
+const ruleMessages = require('../../utils/ruleMessages.cjs');
+const setDeclarationValue = require('../../utils/setDeclarationValue.cjs');
+const properties = require('../../reference/properties.cjs');
+const validateOptions = require('../../utils/validateOptions.cjs');
+
+const ruleName = 'declaration-property-value-keyword-no-deprecated';
+
+const messages = ruleMessages(ruleName, {
+	expected: (actual, expected) => `Expected "${actual}" to be "${expected}"`,
+	rejected: (keyword, property) =>
+		`Unexpected deprecated keyword "${keyword}" for property "${property}"`,
```

Comment:
All rules respect a certain order—for x coming first—except `media-feature-name-unit-allowed-list`.
So Ill conform to our conventions.